test/apps/echoer/main.go

```diff
@@ -15,12 +15,15 @@ func main() {
 	port := os.Getenv("PORT")
 	addr := ":" + port
 
-	if _, err := discoverd.AddServiceAndRegister(service, addr); err != nil {
+	hb, err := discoverd.AddServiceAndRegister(service, addr)
+	if err != nil {
 		log.Fatal(err)
 	}
+	defer hb.Close()
```

Comment:
They're short enough there's no compelling need, and I figured they might be used as examples.  We could convert them; wanna?